integrations/azure-devops/.port/spec.yaml

```diff
@@ -15,6 +15,10 @@ configurations:
     required: true
     type: string
     sensitive: true
+  - name: isProjectAdmin
```

Comment:
I think this name is misleading. Perhaps this should be `isProjectsLimited` or something like this and be set to true by default. 
We shouldn't assume that we have organization-wide permissions.